README.md

```diff
@@ -270,8 +270,9 @@ Calling this method after the response has ended/closed is a NOOP.
 
 Unless you specify a `Content-Length` header yourself, HTTP/1.1 responses
 will automatically use chunked transfer encoding and send the respective header
-(`Transfer-Encoding: chunked`) automatically. If you know the length of your
-body, you MAY specify it like this instead:
+(`Transfer-Encoding: chunked`) automatically. If `Transfer-Encoding` is already given
+the `Content-Length` will be removed.
```

Comment:
You're right. This was already implemented but I missed the documentation about this here. I added  a line in the newest commit. I hope this will enhance the documentation a bit.